frontend/packages/dsco/src/divider/__tests__/Divider.test.tsx

```diff
@@ -0,0 +1,51 @@
+import {render, screen} from '@testing-library/react';
+import {Divider} from '../';
+
+describe('Divider Component', () => {
+  it('renders Divider component', () => {
+    render(<Divider />);
+    expect(screen.getByRole('separator')).toBeInTheDocument();
+  });
+
+  it('applies the primary color class by default', () => {
+    render(<Divider />);
+    const dividerElement = screen.getByRole('separator');
+    expect(dividerElement).toHaveClass('divider-color-primary');
```

Comment:
on dashboard pages we have a linter rule that deprecates us from testing for classNames and we test for some visible/'touchable' attributes and/or behaviours. Also as an option we can try using storybook ui tests here which can help test actual visuals like color

cc @stephenliang should we also add a lint rule to deprecate tests against classNames for frontend monoreppo? I believe it's pretty usefull

Reply:
Ah ok good to know! Should I remove all of the classNames tests and replace it with something like what's below, or should we just use Storybook UI tests and remove all of these tests except the first one (`renders Divider component`)?
``` 
it('renders the primary color by default', () => {
  render(<Divider />);
  const dividerElement = screen.getByRole('separator');
  window.getComputedStyle = jest.fn().mockReturnValue({
    borderTopColor: 'var(--background-neutral-quaternary)',
  });
  const styles = getComputedStyle(dividerElement);
  expect(styles.borderTopColor).toBe('var(--background-neutral-quaternary)');
});
```

Reply:
Both options work for me and in both option we should remove `.toHaveClass ` tests.

If you need some example - here's what I did when I had to acess the styles of Slider component in tests and we didn't have storybook ui tests yet. https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/componentLibrary/SliderTest.tsx#L77

Though, storybook ui tests might end up being more comfortable thing to use for such tests